pkg/controller/provider/web/vsphere/vm.go

```diff
@@ -249,8 +249,13 @@ type VM struct {
 	GuestDisks               []model.GuestDisk    `json:"guestDisks"`
 	GuestIpStacks            []model.GuestIpStack `json:"guestIpStacks"`
 	SecureBoot               bool                 `json:"secureBoot"`
-	DiskEnableUuid           bool                 `json:"diskEnableUuid"`
-	NestedHVEnabled          bool                 `json:"nestedHVEnabled"`
+	ToolsStatus              string               `json:"toolsStatus"`
+	ToolsRunningStatus       string               `json:"toolsRunningStatus"`
```

Comment:
Could we use the `VirtualMachineToolsRunningStatus`, `VirtualMachineToolsStatus` and `VirtualMachineToolsVersionStatus` from govmomi or our own const?